css/styles.css

```diff
@@ -0,0 +1,33 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+body {
+
+}
```

Comment:
normaly you don't leave empty brackets, so you could just delete it, if you don't use it :)
```suggestion

```